### ISSUE TITLE
Quantize model accuracy drops

### ISSUE THREAD

**Issue by OriAlpha** [microsoft/onnxruntime#11286]
**Describe the bug**
I am using Gelectra model which is german language based NLP model. I have converted model to onnx and have done some inference. So the pytorch model was converted to onnx and onnx quantize model. Here onnx model gives same accuracy but with quantize model it results in 5 to 15% accuracy drop. I know its common with quantize model to see accuracy drops. but is there any other ways to optimize the model to have minimal accuracy drop between models. 

code for quantize model 

```
quantize_dynamic(model_input= onnx_model_path,
                 model_output= quantized_onnx_model_path,
                 weight_type=QuantType.QUInt8)
```

**System information**
- OS Platform and Distribution: Linux Ubuntu 20.04
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: 1.10.1
- Python version: 3.9
- CUDA/cuDNN version: 11.3 
- GPU model and memory: RTX A2000 & 4GB

**Comment by yufenglee** [comment]
Could you please try setting option per_channel to True and see if it can get some accuracy back? If it still doesn't meet your goal, you may need to try QAT retraining.

**Comment by OriAlpha** [comment]
I tried per_channel=True, it results in more accuracy drop.

**Comment by yufenglee** [comment]
@OriAlpha, is it possible to share the model, script and data to repro and investigate?

**Comment by OriAlpha** [comment]
Thanks for your interest, but i am not sure i can share the model. But i have used https://huggingface.co/deepset/gelectra-base and done some finetuning for model, You can consider this model to replicate whole process.
